app/models/repo.js

```diff
@@ -130,18 +130,47 @@ Repo.reopen({
     return array;
   }.property(),
 
+  crons: function() {
+    var array, builds, id;
+    id = this.get('id');
+    builds = this.store.filter('build', {
+      event_type: 'cron',
+      repository_id: id
+    }, function(b) {
+      return b.get('repo.id') + '' === id + '' && b.get('eventType') === 'cron';
+    });
+    array = ExpandableRecordArray.create({
+      type: 'build',
+      content: Ember.A([])
+    });
+    array.load(builds);
+    id = this.get('id');
+    array.observe(builds);
+    return array;
+  }.property(),
```

Comment:
Just for the record, I tried `peekAll` approach, but it didn't work for some reason (ie. it wasn't properly updated when new builds were coming in). I didn't have time to look into it further, so we may try it at some point. On the other hand I think that we will be able to use `hasMany` when we have proper pagination metadata from V3, so I'm not sure how much value is in refactoring it now.